lib/puppet/provider/bootstrap_rpm/bootstrap_rpm.rb

```diff
@@ -0,0 +1,139 @@
+require 'fileutils'
+
+Puppet::Type.type(:bootstrap_rpm).provide(:bootstrap_rpm) do
+
+  commands :rpmbuild => 'rpmbuild'
+
+  def create
+    base_dir
+    copy_sources
```

Comment:
```suggestion
    sources_dir
    copy_sources
```

Reply:
Hmm, this should not be needed but it's consistently failing for me in EL8. Perhaps it's calling something after the base_dir has been removed?